lib/autoload/sfAutoload.class.php

```diff
@@ -188,7 +188,7 @@ public function loadClass($class)
     $class = strtolower($class);
 
     // class already exists
-    if (class_exists($class, false) || interface_exists($class, false))
+    if (class_exists($class, false) || interface_exists($class, false) || trait_exists($class, false))
```

Comment:
Since trait aren't supported in PHP 5.3 people trying to use trait on that version will get an error so I guess they won't use it.

It's ok for me if you use `function_exists`.

I won't drop PHP 5.3 mostly because symfony1 is already a legacy project that shouldn't be used for new project but only to maintain old project which needs some small improvment.